agent/tool-scripts/postprocess/gold/prometheus-metrics/stderr

```diff
@@ -0,0 +1 @@
+WARNING:root:not a valid json document, skipping the entry
```

Comment:
prometheus-datalog script generates this error, this occurs when the datalog script gets killed before we the request to the /metrics endpoint is completed which results in last line of the stdout not being a valid json document. So we ignore that particular metric. we also generate a warning to choose appropriate interval.